rim/tests.py

```diff
@@ -1,3 +1,39 @@
-from django.test import TestCase
+from django.test import TestCase, Client
 
 # Create your tests here.
+class TestHttpResponse(TestCase):
+    # Url dictionary to hold names for existing rim web pages, add as neccessary 
+    urlDictionary = {
+        "Admin Panel": '/admin/',
+        "Home": '/',
+        "Add": '/add/',
+        "Edit": '/edit/1',
+        "Client": '/client/1'
```

Comment:
Also this dictionary might have to be moved to the `setUp()` method since it depends on the PKs for equipment and client objects (which aren't guaranteed to always be 1).